docs/knowledge-base.rst

```diff
@@ -0,0 +1,107 @@
+.. _knowleedge-base:
```

Comment:
Would it make sense to move the file to the `topical` subdirectory to mimic the organization of the content?

Reply:
Maybe, then it would also make sense to put tutorial source files into their own folder. But then they would mix up with `files` and `images` folders that have nothing to do with content hierarchy. 

Currently I prefer flat structure, but as soon as number of files goes above ~20 we will need some structure.